templates/register.html

```diff
@@ -259,7 +259,7 @@ <h2 class="sr">${_("Additional Personal Information")}</h2>
           % if settings.REGISTRATION_EXTRA_FIELDS['country'] != 'hidden':
           <li class="field-group">
           <div class="field ${settings.REGISTRATION_EXTRA_FIELDS['country']} select" id="field-country">
-              <label for="country">${_("Country")}</label>
+              <label for="country">${_("Country or Region")}</label>

```

Comment:
Looks like index.html, main.html and navigation.html were updated for Dogwood. Can you also look if register.html should be updated - looks like it's missing some changes.